domain/src/main/java/org/fao/geonet/domain/ISODate.java

```diff
@@ -430,7 +447,8 @@ public void setDateAndTime(String isoDate) {
             // JODAISODate. This class converts to UTC format to avoid timezones
             // issues.
             String afterT = timeAndDate.substring(indexOfT + 1);
-            boolean timeZoneInfo = afterT.contains("+") || afterT.contains("-");
+            boolean timeZoneInfo = afterT.contains("+") || afterT.contains("-")
+                || afterT.endsWith("Z") || afterT.endsWith("Z");
```

Comment:
Any reason to check the same thing twice?

Reply:
Sorry one of those was intended to be lowercase.